AntonBabunin/src/main/java/ua/artcode/market/Interfaces/ITerminal.java

```diff
@@ -0,0 +1,10 @@
+package ua.artcode.market.Interfaces;
+
+import ua.artcode.market.bill.Bill;
+
+public interface ITerminal {
+
+    Bill createBill (Object object);
```

Comment:
change object to Product or something else

Reply:
Why? It implemented in two classes: Terminal and Salesman

Reply:
You should understand structure. Models it`s classes without any logic. Only fields. And Some Controllers that use created models, For example TerminalController which will implement ITerminal. Who can create Bill? Terminal or Salesman

Reply:
Salerman using a terminal :)